src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,32 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.IncorrectInputDataException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_LEN_FOR_CREDS = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user.getAge() == null || user.getLogin() == null
+                || user.getLogin().isEmpty() || user.getId() == null
+                || user.getId() < 0 || user.getPassword() == null) {
+            throw new IncorrectInputDataException("Some of the field is empty");
```

Comment:
This is a bad approach to test fields for null value in a single if statement, we should throw an exception message to the client which field is null or empty.